### PR TITLE
Documented each addition to Velocity 3.1.2/3.2.0 API

### PR DESCRIPTION
**Pull request by 4drian3d**
Progress on #744

### CODE REVIEW

File: api/src/main/java/com/velocitypowered/api/proxy/crypto/KeySigned.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 import org.checkerframework.checker.nullness.qual.Nullable;
 
+/**
+ * @since 3.1.2
+ */
```
Comment (BBaoVanC) [inline api/src/main/java/com/velocitypowered/api/proxy/crypto/KeySigned.java]: Checkstyle says this needs a summary

File: api/src/main/java/com/velocitypowered/api/proxy/crypto/SignedMessage.java [hunk 2]
```diff
@@ -9,6 +9,9 @@
 
 import java.util.UUID;
 
+/**
+ * @since 3.1.2
+ */
```
Comment (BBaoVanC) [inline api/src/main/java/com/velocitypowered/api/proxy/crypto/SignedMessage.java]: this one needs summary too

**Review by BBaoVanC — COMMENTED** [review]
This PR fails to build because of checkstyle ("Summary javadoc is missing.")

image

**Comment by 4drian3d** [comment]
I solved the compilation errors and added the annotation in all the places I found since last time.
Fortunately the only addition to the API in version 3.2.0 so far is properly documented.

**Pull request closed without merging**